### DISCUSSION TITLE
Allow empty x-init

### DISCUSSION THREAD

**Discussion by stefanodelpero** [alpinejs/alpine#1991] [3. Feature ideas]
Currently and empty `x-init` cause the error `Uncaught SyntaxError: expected expression, got '}'`.
It would be useful to be able to have an empty `x-init` when, server-side, its content is dynamic.
So one could write `<div x-init="@initContent">` and do not have to check if it is empty.

**Comment by MuzafferDede** [comment]
hmm, let me take a look at this one.

**Reply by MuzafferDede** [reply]
- #1993 should fix this!

**Comment by stefanodelpero** [comment]
Perfect! Thank you!!!
